scripts/publish-plugin.sh

```diff
@@ -17,7 +17,7 @@ if [ "$(go env GOOS)" = "windows" ]; then
 fi
 
 (cd "${ROOT}/provider" && go build \
-   -ldflags "-X github.com/pulumi/pulumi-azure/provider/pkg/version.Version=${VERSION}" \
+   -ldflags "-X github.com/pulumi/pulumi-azure/provider/pkg/v2/version.Version=${VERSION}" \
    -o "${WORK_PATH}/pulumi-resource-azure${BIN_SUFFIX}" \
    "${ROOT}/cmd/pulumi-resource-azure")
```

Comment:
Does this same thing apply here with the package being updated to "${ROOT}/cmd/pulumi-resource-azure?

Reply:
yes it will - you are correct

Reply:
actually no, that *is* a path as it;s a go build

```
▶ ./scripts/publish-plugin.sh
can't load package: package ./v2/cmd/pulumi-resource-azure: cannot find package "." in:
	/Users/stack72/code/go/src/github.com/pulumi/pulumi-azure/provider/v2/cmd/pulumi-resource-azure
```